internal/cloud/openssh/client.go

```diff
@@ -73,3 +82,41 @@ func destination(username, hostname string) string {
 
 	return result
 }
+
+func logCmdRun(cmd *exec.Cmd) error {
+	// Use cmd.Start so we can log the pid. Don't bother writing errors to
+	// the debug log since those will be printed anyway.
+	if err := cmd.Start(); err != nil {
+		return fmt.Errorf("openssh: start command %q: %w", cmd, err)
+	}
+	debug.Log("openssh: started process %d with command %q", cmd.Process.Pid, cmd)
```

Comment:
no objection, more curiosity: why did you feel the need to log the pid? what is the pid used for?

Reply:
Just to help me track down which instance of SSH was causing problems when I had multiple connections running.